### PR TITLE
Create Beyond06-SOW.md for next course with updated dates

### PR DESCRIPTION
**Pull request by mgold1234**

### CODE REVIEW

File: Beyond06-SOW.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+| 1              | 17.10                  | Intro session (housekeeping) ~50 min +Talk about initial challenges and student selection                                                                                                                                                                    | Liora Milbaum        |
+|                |                        | Modern software applications - Architecture, Tooling and development processes.                                                                                                                                                                              | Barak Korren         |
+|                |                        | Incorporate presenting of the 1st exercise and the skill evaluation form in the 1st lecture                                                                                                                                                                  | Barak Korren         |
```
Comment (ifireball) [inline Beyond06-SOW.md:167]: The above line is probably not something the student should see - it looks like a task I wrote for myself...

File: Beyond06-SOW.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+|----------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| -              | 10.10                   | Self learning: Tool introduction challenge                                                                                                                                                                                                                   | (Self work)          |
+| 1              | 17.10                  | Intro session (housekeeping) ~50 min +Talk about initial challenges and student selection                                                                                                                                                                    | Liora Milbaum        |
```
Comment (ifireball) [inline Beyond06-SOW.md:165]: why does it say 17/0 here? The intro session is on 10/10...

File: Beyond06-SOW.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+|                |                        | Incorporate presenting of the 1st exercise and the skill evaluation form in the 1st lecture                                                                                                                                                                  | Barak Korren         |
+| -              | 15.10                  | First challenge   self evaluation form submission date                                                                                                                                                                                                       | (Self work)          |
+| 2              | 31.10                  | Team collaboration: Git, GitHub, PRs and issues.                                                                                                                                                                                                             | Liora Milbaum        |
```
Comment (ifireball) [inline Beyond06-SOW.md:169]: We don't have a Git/GitHub lecture this time, did you update this table at all?

File: Beyond06-SOW.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+| -              |                  | Git challenge submission date                                                                                                                                                                                                                                | (Self work)          |
+| 3              |                  | Advanced Git and GitHub skills                                                                                                                                                                                                                               | Liora Mibaum         |
+| 4              | 07.11                  | Demo 1:  Pre-requirements  fully working env (PRs, vagrant, issues with dependencies) Student 1: vagrant env Student 2: Django Student 3: Home page Student 4: CI with flake8 Student 5: ReadMe page (in git)  Demo: Guided Demo: Project presentation, MVP  |                      |
```
Comment (ifireball) [inline Beyond06-SOW.md:175]: No need to write the full requirements in this table, they are specified above already

File: Beyond06-SOW.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+* Grade 4 - Review teammates PRs, the leader(s) of the group (the most engaged teammate(s) in others' tasks), helps others.
+
+Execution process quality - in this section we will measure the github activity (PRs) and the improvement in the technology stuck (frontend, backend, Data model, tests). We would like to see feature planning and the ability to learn new areas despite failures.
+
+* Grade 1 - not enough PRs, not involved in feature planning, was involved with only 1 layer from the technology stack(frontend, backend, Data model, tests).
+* Grade 2 - a few merged PRs ( minor PRs or 1-2 PR for a demo period) , involving only 2 layers of the technology stack(frontend, backend, DBs, tests).
+* Grade 3 - Wrote PRs involving at least 3 of the 4 layers in the technology stuck (frontend, backend, Data model, tests), part of feature planning, for each PR attached an issue.
+* Grade 4 - touching on all the technology stack (frontend, backend, Data model, tests), every week publish and merge at least 2 significant PRs, each PR includes tests, ability to learn new areas and subject on himself.
+
+## Grade Process
```
Comment (ifireball) [inline Beyond06-SOW.md:211]: We already talk about the performance review above. We can probably just drop the res of this section as it does not correctly describe the actual grading process.

**Review by ifireball — CHANGES_REQUESTED** [review]
Overall good work, but it does seem there are more sections that need to be updated.

**Review by mgold1234 — COMMENTED** [review]
fix all comments

File: Beyond06-SOW.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+
+## Division into project groups (till Oct22th)
+
+After the first 2 weeks you'll be divided into 4 groups.
+
+Each group will have mentors:
+
+* group1: Yariv, Arie, Shimrit & Omer
+* group2: Kobi, Michal, Nati & Nahshon
+* group3: Barak, Daniel, Guy & Eran
```
Comment (ifireball) [inline Beyond06-SOW.md:151]: Missing group 4?

File: Beyond06-SOW.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+
+## Syllabus (list of sessions)
+
+| Session number | Date                   | Topic(s)                                                                                                                                                                                                                                                     | Lecturer             |
+|----------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|                                                                                                     | 1              | 10.10                  | Intro session (housekeeping) ~50 min +Talk about initial challenges and student selection                                                                                                                                                                    | Liora Milbaum        |
+|                |                        | Self learning: Tool introduction challenge
```
Comment (ifireball) [inline Beyond06-SOW.md:165]: I think mentioning the 1st challenge both here and in the submission data might be confusing - lets either stick to how I wrote it in the spreadsheet (hand-off v.s. submission) or specify only the submission dates.

File: Beyond06-SOW.md [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+                                                                                      | (Self work)          |  
+
+                                           Modern software applications - Architecture, Tooling and development processes.                                                                                                                                                                              | Barak Korren         |
+
+| -              | 15.10                  | First challenge   self evaluation form submission date                                                                                                                                                                                                       | (Self work)          |
+| 2              | 17.10                  | Python 101                                                                                                                                                                                                                                                   |  Kobi Hakimi         |
```
Comment (ifireball) [inline Beyond06-SOW.md:171]: Missing the submission date for the 2nd challenge here.

File: Beyond06-SOW.md [hunk 9]
```diff
@@ -0,0 +1,344 @@
...
+| 8              | 5.12                   | Going to production 1 - PAAS and containers                                                                                                                                                                                                                  | Barak Korren         |
+
+|                |                        | Writing and testing the Django application views and templates, bootstrapping (navigation), Practicing testing                                                                                                                                               |  Yariv Rachmany      |
+| 9              | 19.12                  | Demo 3: Testing, Control logic and UI                                                                                                                                                                                                                        |                      |
```
Comment (ifireball) [inline Beyond06-SOW.md:182]: Lets keep the demo names consistent between here, the list above and the spreadsheet.

Same goes for lecture names

**Review by ifireball — CHANGES_REQUESTED** [review]

File: Beyond06-SOW.md (hunk 6 above)
Comment (ifireball) [reply Beyond06-SOW.md:151]: Not seeing the 4th group yet

File: Beyond06-SOW.md (hunk 7 above)
Comment (ifireball) [reply Beyond06-SOW.md:165]: Not seeing any change in the file about this

File: Beyond06-SOW.md (hunk 8 above)
Comment (ifireball) [reply Beyond06-SOW.md:171]: Still missing...

File: Beyond06-SOW.md (hunk 9 above)
Comment (ifireball) [reply Beyond06-SOW.md:182]: I meant for the whole list and table, not just this line - in any case, not seeing the change you've mentioned

File: Beyond06-SOW.md (hunk 5 above)
Comment (ifireball) [reply Beyond06-SOW.md:211]: Still seeing the section here...

**Review by ifireball — CHANGES_REQUESTED** [review]

**Pull request closed without merging**